gef.py

```diff
@@ -3971,12 +3981,12 @@ def __init__(self, set_func, vbp_num, addr):
         self.bp_addr = 0
         # this address might be a symbol, just to know where to break
         if isinstance(addr, int):
-            self.addr = hex(addr)
+            self.addr: Union[int, str] = hex(addr)
```

Comment:
I wonder whether we should use an `Address` type here.

Reply:
added this thought to the linked project 11